test/suite/test_config08.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-2020 MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# test_config08.py
+#   Test the configuration that enables/disables dirty tree flushing.
+#
+
+import time, wiredtiger, wttest
+from wtscenario import make_scenarios
+
+class test_config08(wttest.WiredTigerTestCase):
+
+    logging = [
+        ('log_off', dict(log='(enabled=false)')),
+        ('log_on', dict(log='(enabled=true)')),
+    ]
+    fileFlush = [
+        ('flush_off', dict(file_close_sync='false')),
+        ('flush_on', dict(file_close_sync='true')),
+    ]
+
+    scenarios = make_scenarios(logging, fileFlush)
+
+    # This test varies the logging and file flush settings and therefore needs to set up its own
+    # connection. Override the standard methods.
+    def setUpConnectionOpen(self, dir):
+        return None
+    def setUpSessionOpen(self, conn):
+        return None
+    def ConnectionOpen(self, log, file_close_sync):
+        self.home = '.'
+        # Explicitly increase time between checkpoints
+        conn_params = 'create,checkpoint=(wait=60),' + \
+            'log=' + log + ',' + 'file_close_sync=' + file_close_sync
+        try:
+            self.conn = wiredtiger.wiredtiger_open(self.home, conn_params)
+            self.session = self.conn.open_session()
+        except wiredtiger.WiredTigerError as e:
+            print("Failed conn at '%s' with config '%s'" % (dir, conn_params))
+
+    # Create a table with logging setting matching the connection level config.
+    def test_config08(self):
+        self.ConnectionOpen(self.log, self.file_close_sync)
```

Comment:
You can remove this with that function above.

Reply:
I copy-pasted the boiler plate code from another test because I didn't know how to change default connection config. But looks like overriding the `conn_config()` is a much cleaner approach.